### DISCUSSION TITLE
TrueNAS 'core' or 'scale'

### DISCUSSION THREAD

**Discussion by isopix** [TheGrandWazoo/freenas-proxmox#169] [Q&A]
I wonder if this plugin works equally well with both 'scale' and 'core' versions of TrueNAS?

**Comment by eugenefvdm** [comment]
The short answer is in my opinion yes it works equally well with both versions.

iSCSI is a standard interface technology for networked disk access.

Both `core` and `scale` supports iSCSI which in turn uses standard Linux/Unix libraries.

TheGrandWazoo's plugin assists with getting `core` and `scale` interfacing with iSCSI which can be a mission. The mission part is creating LUNs, extents, resizing disks etc. since Proxmox doesn't have this built in and you'd normally have to do this manually. This plugin uses the FreeNAS APIs to do this interfacing and both `core` and `scale` and ancestors of FreeNAS.

**Comment by TheGrandWazoo** [comment]
Yes. I have my cluster attached to both a TrueNAS-Core and TrueNAS-Scale servers without any issues.

**Reply by Dennis-Ayala** [reply]
Which versions?
